lib/Conversion/TorchToLinalg/DataMovement.cpp

```diff
@@ -284,15 +284,25 @@ class ConvertAtenViewOp : public OpConversionPattern<AtenViewOp> {
   // `aten.view`.
   static std::pair<SmallVector<int64_t>, SmallVector<int64_t>>
   getInputAndOutputShape(Value inputTorchTensor,
```

Comment:
Are the changes here necessary? (from doing some experimenting locally, the handling of leading dims seems to work without this)

Reply:
This was actually the easier of the two problems we had to solve, and I just forgot to mention we fixed it, but the idea here is that if we have information about the resultType already, we should use that to calculate outputDims. I'll get an example to show